assets/src/scripts/monitoring-location/components/hydrograph/selectors/time-series-data.js

```diff
@@ -77,3 +77,33 @@ export const getPrimaryParameterUnitCode = createSelector(
     getPrimaryParameter,
     parameter => parameter ? parameter.unit : null
 );
+
+/*
+ * Returns a Redux select which returns the IV data method id with the most points. If more than
+ * one method has the same point count, then the time series with the most recent point is chosen.
+ * @returns {Function} which returns {String} the preferred method ID.
+ */
+export const getPreferredIVMethodID = createSelector(
```

Comment:
I decided to put this bit of logic in a selector rather than in the method-picker module. It allows us to more completely and easily test the logic.